includes/payments/actions.php

```diff
@@ -253,3 +253,149 @@ function give_refresh_thismonth_stat_transients( $payment_ID ) {
 }
 
 add_action( 'save_post_give_payment', 'give_refresh_thismonth_stat_transients' );
+
+
+/**
+ * Add support to get all payment meta.
+ * Note: function for internal use
+ *
+ * @since 2.0
+ *
+ * @param $check
+ * @param $object_id
+ * @param $meta_key
+ * @param $meta_value
+ *
+ * @return array
+ */
+function __give_get_payment_meta( $check, $object_id, $meta_key, $meta_value ) {
```

Comment:
@DevinWalker We can still get all payment meta after running the update. It will help us to save queries. Some keys formatted while others are same.